<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IWoo_Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
background-color: rgb(34, 34, 34);

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QToolButton" name="btn_play">
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-color: transparent;

}</string>
      </property>
      <property name="text">
       <string>|&gt;</string>
      </property>
      <property name="icon">
       <iconset resource="resourses.qrc">
        <normaloff>:/img/pl.png</normaloff>:/img/pl.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="btn_pause">
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-color: transparent;

}
</string>
      </property>
      <property name="text">
       <string>||</string>
      </property>
      <property name="icon">
       <iconset resource="resourses.qrc">
        <normaloff>:/img/ps.png</normaloff>:/img/ps.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="btn_previous">
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-color: transparent;

}
</string>
      </property>
      <property name="text">
       <string>|&lt;</string>
      </property>
      <property name="icon">
       <iconset resource="resourses.qrc">
        <normaloff>:/img/t.png</normaloff>:/img/t.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="btn_add">
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-color: transparent;

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resourses.qrc">
        <normaloff>:/img/g+.png</normaloff>:/img/g+.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QToolButton" name="btn_next">
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-color: transparent;

}</string>
      </property>
      <property name="text">
       <string>&gt;|</string>
      </property>
      <property name="icon">
       <iconset resource="resourses.qrc">
        <normaloff>:/img/gt.png</normaloff>:/img/gt.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QSlider" name="sl_volume">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider{
	selection-background-color: rgb(62, 62, 62);
	selection-color: rgb(31, 31, 31);
	gridline-color: rgb(38, 38, 38);
background-color: rgb(34, 34, 34);
	
}
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="8">
     <widget class="QLabel" name="currentTrack">
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QToolButton" name="btn_stop">
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-color: transparent;

}
</string>
      </property>
      <property name="text">
       <string>S</string>
      </property>
      <property name="icon">
       <iconset resource="resourses.qrc">
        <normaloff>:/img/st.png</normaloff>:/img/st.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="time">
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="l_volume">
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: rgb(255, 255, 255);

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QSlider" name="h_slider">
      <property name="styleSheet">
       <string notr="true">QSlider{
background-color: rgb(34, 34, 34);

}
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="9">
     <widget class="QTableView" name="playlistView">
      <property name="styleSheet">
       <string notr="true">QTableView{
	alternate-background-color: rgb(30, 30, 30);
	color: rgb(255, 255, 255);
	background-color: rgb(32,32, 32);
	border-color: rgb(0, 0, 0);
	border-top-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
	border-left-color: rgb(0, 0, 0);
	border-bottom-color: rgb(0, 0, 0);

}</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="9">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(60, 60, 60);

}</string>
      </property>
      <property name="text">
       <string>Очистить плейлист</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
